pkg/kubelet/container/cache_test.go

```diff
@@ -34,6 +34,8 @@ func newTestCache() *cache {
 func TestCacheNotInitialized(t *testing.T) {
 	cache := newTestCache()
 	// If the global timestamp is not set, always return nil.
+	cache.lock.Lock()
```

Comment:
Here `cache` only got one consumer. I don't think acquiring the lock is needed.

Reply:
@dixudx actually, I had the same feeling about this. but considering what test case does, we should follow the principles to test the units as much as we can IMHO :)